auth-web/src/views/auth/PriceListView.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <main class="view-container container">
+      <h1 class="mb-4">Price List</h1>
+      <p class="mb-8">Where applicable, tax is included in the price shown. Prices are subject to change without notice.</p>
+      <section class="mb-10">
+        <h2 class="mb-5">Business Registry</h2>
+        <v-card flat>
+          <table>
+            <thead>
+              <th class="product text-left">Product</th>
+              <th class="fee">Price</th>
+              <th class="fee--service">Service Fee</th>
+              <th class="fee--priority">Priority Fee</th>
+              <th class="fee--future">Future Effective Fee</th>
+            </thead>
+            <tbody>
+              <tr v-for="item in pricelist" :key="item.name">
+                <td class="product font-weight-bold">{{item.name}}</td>
+                <td class="fee">
+                  <span aria-describedby="Annual Report PRice">{{'$' + item.price}}</span>
+                </td>
+                <td class="fee--service">{{'$' + item.serviceFee}}</td>
+                <td class="fee--priority"><span v-if="item.priorityFee">{{'$' + item.priorityFee}}</span></td>
+                <td class="fee--future"><span v-if="item.futureEffectiveFee">{{'$' + item.futureEffectiveFee}}</span></td>
+              </tr>
+            </tbody>
+          </table>
+        </v-card>
+      </section>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { VueConstructor } from 'vue'
+
+@Component({
+  name: 'PriceListView'
+})
+
+export default class PriceListView extends Vue {
+  data () {
```

Comment:
No need of data return. we can use private pricelist:

Reply:
Updated as requested.